lib/Entities/SegmentEntity.php

```diff
@@ -109,4 +109,8 @@ public function getDynamicFilters() {
   public function isDynamic(): bool {
     return $this->getType() === self::TYPE_DYNAMIC;
   }
+
+  public function isStatic(): bool {
```

Comment:
I am not sure why do we need this.

Reply:
This condition is used in [SubscriberFinder](https://github.com/mailpoet/mailpoet/blob/master/lib/Segments/SubscribersFinder.php#L53) and I added the same one to a different place so I put it in this method so that it is defined on one place.